src/Comment/Commentable.php

```diff
@@ -7,17 +7,17 @@
 interface Commentable
 {
     /**
-     * @param array<array-key, Comment> $comments
+     * @param array<int<0, max>, Comment> $comments
```

Comment:
Can this be a `list` after #1173 is merged?

Reply:
Probably. I'll recheck once #1173 is merged.